File: Auto-Schedule/Domain/Model/ManualScheduleModel.cs
using Domain.Enum;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Model
{
    public class ManualScheduleModel
    {
        public Guid? Id { get; set; }
        public String Day { get; set; }
        public string StartTime { get; set; }
        public string EndTime { get; set; }
        public Guid CourseLecturesId { get; set; }
        public Guid HallsId { get; set; }
        public Guid LocationId { get; set; }
        public Guid DepartmentId { get; set; }
        public Guid GroupId { get; set; }
        //public Guid? UserId { get; set; }
    }
}

File: frontend/src/Interfaces/ManualScheduleModel.ts
export interface ManualScheduleModel
{
    id: string | null,
    day: string,
    startTime: string,
    endTime: string,
    courseLecturesId: string,
    hallsId: string,
    locationId: string,
    departmentId: string,
    groupId: string;
    hasReport: string;
    isCanceled: string;
}